### PR TITLE
Translate views to hybrid Polymer 1 and 2

### PR DESCRIPTION
**Pull request by warpech**
Implements https://github.com/Starcounter/Website/issues/297

### CODE REVIEW

**Comment by warpech** [comment]
This is a requirement for #307. 

@alshakero could you please review it on Thursday morning?

File: src/WebsiteEditor/wwwroot/WebsiteEditor/views/RoutingPage.html [hunk 1]
```diff
@@ -137,14 +133,14 @@ <h5>Choose surface for route</h5>
...
                 <button class="btn btn-xs btn-default" dialog-dismiss value="{{model.CancelModalTrigger$::click}}" onmousedown="++this.value;">Cancel</button>
                 <span>&nbsp;</span>
                 <button class="btn btn-xs btn-success" title="Add new route" value="{{model.CreateModalTrigger$::click}}" onmousedown="++this.value;">Create route</button>
             </div>
         </paper-dialog>
-    </template>
+    </template></dom-bind>
     <script>
         (function () {
             var template = document.currentScript.previousElementSibling;
```
Comment (alshakero) [inline src/WebsiteEditor/wwwroot/WebsiteEditor/views/RoutingPage.html]: I think this should be:

Comment (alshakero) [inline src/WebsiteEditor/wwwroot/WebsiteEditor/views/RoutingPage.html]: I think this should be:
```js
var script = document._currentScript || document.currentScript;
var template = Polymer.Element ? script.previousElementSibling : script.previousElementSibling.firstElementChild;
```

File: src/WebsiteEditor/wwwroot/WebsiteEditor/views/BlendingPointsPage.html [hunk 2]
```diff
@@ -93,7 +90,7 @@
                 </div>
             </paper-collapse-item>
         </template>
-    </template>
+    </template></dom-bind>
     <script>
         (function () {
             var template = document.currentScript.previousElementSibling;
```
Comment (alshakero) [inline src/WebsiteEditor/wwwroot/WebsiteEditor/views/BlendingPointsPage.html]: This should be:
```js
var script = document._currentScript || document.currentScript;
var template = Polymer.Element ? script.previousElementSibling : script.previousElementSibling.firstElementChild;
```

**Comment by warpech** [comment]
Thanks so much for the careful review! Again my bad for only checking `document.currentScript` :/

**Comment by warpech** [comment]
Setting as “Finished” because it is:
- released (with release notes) https://github.com/Starcounter/Website/releases/tag/4.2.0

**Pull request merged**